### PR TITLE
AclExtras Windows compatible

### PR DESCRIPTION
**Pull request by geraldurbas**
corrected path usage with DS constant

### CODE REVIEW

File: src/AclExtras.php [hunk 1]
```diff
@@ -178,10 +178,10 @@ protected function _updateControllers($root, $controllers, $plugin = null)
         // look at each controller
         $controllersNames = [];
         foreach ($controllers as $controller) {
-            $tmp = explode('/', $controller);
+            $tmp = explode(DS, $controller);
             $controllerName = str_replace('Controller.php', '', array_pop($tmp));
             $controllersNames[] = $controllerName;
-            $path = $this->rootNode . '/' . $pluginPath . $controllerName;
+            $path = $this->rootNode . DS . $pluginPath . $controllerName;
```
Comment (ceeram) [inline src/AclExtras.php]: Do we really need to change this too? isnt this the path in the db? Changing that per environment would mean you cannot move the db to another OS without breaking the application

File: src/AclExtras.php [hunk 2]
```diff
@@ -366,6 +366,7 @@ public function recover()
      */
     protected function _getNamespace($className, $pluginPath = false)
     {
+		$className = str_replace (DS, '/', $className);
```
Comment (ceeram) [inline src/AclExtras.php]: Indentation is wrong here

File: src/AclExtras.php (hunk 1 above)
Comment (dereuromark) [inline src/AclExtras.php]: I agree with @ceeram . Not necessarily needed. Both slashes work fine and would make the DB cross OS save if it stayed.

File: src/AclExtras.php (hunk 2 above)
Comment (rchavik) [inline src/AclExtras.php]: @dereuromark

I guess only this line is needed?

Comment (dereuromark) [inline src/AclExtras.php]: Yes

**Comment by ceeram** [comment]
Seems the comments have not been taken care of and the creator of this PR removed his fork.

File: composer.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-	"name": "cakephp/acl",
+	"name": "geraldurbas/acl.git",
```
Comment (ceeram) [inline composer.json]: this should not be changed

File: composer.json [hunk 4]
```diff
@@ -15,11 +15,11 @@
 		"issues": "https://github.com/cakephp/acl/issues",
 		"forum": "http://stackoverflow.com/tags/cakephp",
 		"irc": "irc://irc.freenode.org/cakephp",
-		"source": "https://github.com/cakephp/acl"
+		"source": "https://github.com/geraldurbas/acl.git"
```
Comment (ceeram) [inline composer.json]: This should also not be changed

**Comment by rchavik** [comment]
Sorry, 

I should have closed this since this has been cherrypicked with modification in 9f4da3586.

**Pull request closed without merging**